apps/demos/Demos/Chat/AIAndChatbotIntegration/Vue/App.vue

```diff
@@ -76,6 +76,8 @@ const copyButtonIcon = ref('copy');
 
 onBeforeMount(() => {
   loadMessages(dictionary);
+
+  chatService = new AzureOpenAI(AzureOpenAIConfig);
```

Comment:
I've checked this change on az-server and seems like it does not resolve the issue.
However, replacing current import with `import openai from 'openai';` helps.
So I suggest you to revert this change but update the import

Reply:
[b83ec79eabe4b3dfb5b357bd0ea9b4bfe39a0859]